src/Bootstrap/Configurator.php

```diff
@@ -112,6 +115,17 @@ public function addParameters(array $params)
 
 
 	/**
+	 * Add instances of services.
+	 * @return self
+	 */
+	public function addServices(array $services)
+	{
+		$this->services = DI\Config\Helpers::merge($services, $this->services);
+		return $this;
```

Comment:
I think at least `array_merge` should be used, because if not, old services would not be overwritten by successive calls, which is expected (imho).